### PR TITLE
Update msg_hash_pl.h

### PR DESCRIPTION
**Pull request by DEX357**
## Guidelines

1. Rebase before opening a pull request
2. If you are sending several unrelated fixes or features, use a branch and a separate pull request for each
3. If possible try squashing everything in a single commit. This is particularly beneficial in the case of feature merges since it allows easy bisecting when a problem arises

## Description

[Description of the pull request, detail any issues you are fixing or any features you are implementing]

## Related Issues

[Any issues this pull request may be addressing]

## Related Pull Requests

[Any other PRs from related repositories that might be needed for this pull request to work]

## Reviewers

[If possible @mention all the people that should review your pull request]

### CODE REVIEW

**Comment by DEX357** [comment]
Can anyone check the error of accepting  file?

**Comment by DEX357** [comment]
Can anyone check the error of accepting the file?

File: intl/msg_hash_pl.h [hunk 1]
```diff
@@ -3830,9 +3830,202 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_MENU_XMB_ANIMATION_OPENING_MAIN_MENU,
...
+    )
+   MSG_HASH(
+    MENU_ENUM_LABEL_VALUE_MENU_SHOW_QUIT_RETROARCH,
+    "Pokaż Uruchom ponownie RetroArch"
+    )
+MSG_HASH(
+    MENU_ENUM_SUBLABEL_MENU_SHOW_QUIT_RETROARCH,
+    "Pokaż/ukryj opcję „Uruchom ponownie RetroArch”."
+    )
+#else
```
Comment (orbea) [inline intl/msg_hash_pl.h]: Try removing this line, this is the relevant error.

```
In file included from intl/msg_hash_pl.c:47:
intl/msg_hash_pl.h:3871:2: error: #else without #if
 #else
  ^~~~
```
I would also check where you copy pasted it from and see if you missed something?

Comment (DEX357) [reply intl/msg_hash_pl.h]: hmm but else is in US file

Comment (orbea) [reply intl/msg_hash_pl.h]: Look for the matching `#if` or `#ifdef` and `#endif` in the us file.

**Comment by DEX357** [comment]
fixplup

**Pull request closed without merging**